proposals/0259-DisabledSoftbuttons.md

```diff
@@ -23,21 +27,21 @@ For custom softbuttons, add a parameter to `SoftButtonCapabilities` to let the a
     .
     .
     .
-    <param name="disabledButtonPossible" type="Boolean" mandatory="true">
+    <param name="supportsDisabled" type="Boolean" mandatory="true">
         <description>The button supports being disabled. If the button is set to disabled, it will still show on the HMI but appear grayed out.
         </description>
     </param>
 </struct>
 ```
 
-Plus add an `isEnabled` flag to `SoftButton`. If the flag is true or missing, the softbutton must appear usable. If false, softbutton must appear as not usable. If the button is set to disabled, it will still show on the HMI but appear grayed out. Button press events will still be sent to the app.  Button press events would still be sent to the app so the app can enable experiences like "Repeat is for Premium customers only". Alternatively we could have the button not send any button events to match what Android and iOS do when a button is on screen but disabled.
+Plus add an `isDisabled` flag to `SoftButton`. If the flag is false or missing, the softbutton must appear usable. If true, softbutton must appear as not usable. If the button is set to disabled, it will still show on the HMI but appear grayed out. Button press events will still be sent to the app.  Button press events would still be sent to the app so the app can enable experiences like "Repeat is for Premium customers only". If a button is set to disabled and highlighted, it's up to the HMI to depict the best possible experience.
```

Comment:
```suggestion
Plus add an `isDisabled` flag to `SoftButton`. If the flag is false or missing, the softbutton must appear usable. If true, the `SoftButton` will still show on the HMI but appear grayed out or otherwise disabled. Button press events will still be sent to the app so the app can enable experiences like "Repeat is for Premium customers only". If a button is set to disabled and highlighted, it's up to the HMI to depict the best possible UI.
```